### PR TITLE
BREWs 17, 693 and 704

### PR DESCRIPTION
**Pull request by revilowaldow**

### CODE REVIEW

File: adventure/Kobold Press; Warlock Lair 50 The Light of Memoria.json [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+			],
+			"savingThrowForced": [
+				"wisdom"
+			],
+			"fluff": {
+				"_monsterFluff": {
+					"name": "Spectral Memory",
+					"source": "WL50"
+				}
+			}
```
Comment (Spappz) [inline adventure/Kobold Press; Warlock Lair 50 The Light of Memoria.json:166]:
```suggestion
			},
			"hasFluff": true
```

**Review by Spappz — CHANGES_REQUESTED** [review]

File: adventure/Kobold Press; Warlock Lair 50 The Light of Memoria.json (hunk 1 above)
Comment (revilowaldow) [reply adventure/Kobold Press; Warlock Lair 50 The Light of Memoria.json:166]: 🤡

Comment (Spappz) [reply adventure/Kobold Press; Warlock Lair 50 The Light of Memoria.json:166]: dw i'm running my script again and there's a lot of other files needing it 😏

Comment (revilowaldow) [reply adventure/Kobold Press; Warlock Lair 50 The Light of Memoria.json:235]: https://github.com/TheGiddyLimit/homebrew/pull/1978/commits/64a39607fb8a8f69b6f70dff40943b9731b816c6
Resolved

**Review by TheGiddyLimit — APPROVED** [review]

**Pull request merged**